tests/ui/consts/const-eval/ub-enum.32bit.stderr

```diff
@@ -108,13 +108,13 @@ error[E0080]: evaluation of constant value failed
   --> $DIR/ub-enum.rs:96:77
    |
 LL | const BAD_UNINHABITED_WITH_DATA1: Result<(i32, Never), (i32, !)> = unsafe { mem::transmute(0u64) };
-   |                                                                             ^^^^^^^^^^^^^^^^^^^^ transmuting to uninhabited type
+   |                                                                             ^^^^^^^^^^^^^^^^^^^^ constructing invalid value at .<enum-variant(Ok)>.0.1: encountered a value of uninhabited type Never
```

Comment:
Ok, after rebasing this now gets the nicer value validity error from #109435 instead of the generic "entering unreachable code" it [had before](https://github.com/rust-lang/rust/commit/19a4844f9d49367285af0d2a5d7d8f08dabfa3ce#diff-d43f3390655f1d19dddd720d06998ec6a4f5a55c860d435170efe99b7603de90L111) 🎉